packages/plugin-ext/src/hosted/node/plugin-host-rpc.ts

```diff
@@ -197,7 +199,8 @@ export class PluginHostRPC {
                 for (const api of extApi) {
                     if (api.backendInitPath) {
                         try {
-                            const extApiInit = require(api.backendInitPath);
+                            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                            const extApiInit: any = dynamicRequire(api.backendInitPath);
```

Comment:
Nitpick: we could use `{ provideApi: ExtPluginApiBackendInitializationFn }` or `dynamicRequire<{ provideApi: ExtPluginApiBackendInitializationFn }>` instead of `any` here.